go.mod

```diff
@@ -84,3 +84,5 @@ replace git.schwanenlied.me/yawning/bsaes.git => github.com/Yawning/bsaes v0.0.0
 // If you change this please also update .github/pull_request_template.md and
 // docs/INSTALL.md.
 go 1.15
+
+replace github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt v3.2.1+incompatible
```

Comment:
nit: move replace directive with other ones, above the version number? Same in the kvdb go.mod file.

Reply:
I moved it and also included a comment. Thanks